File: practice/beginner/TAXSAVING/taxsaving.cc
// https://www.codechef.com/problems/TAXSAVING

/* SPDX-FileCopyrightText: 2023 Vladimir Rusinov <vladimir.rusinov@gmail.com>
 *
 * SPDX-License-Identifier: Apache-2.0
 */

#include <iostream>
using namespace std;

int tax_saving(int x, int y) {
    return x - y;
}

int main() {
	int t;
    cin >> t;
    for (int i = 0; i < t; i++) {
        int x, y;
        cin >> x;
        cin >> y;
        cout << tax_saving(x, y) << "\n";
    }
	return 0;
}

File: practice/beginner/500_diff/TAXSAVING/taxsaving.py
# SPDX-FileCopyrightText: 2025 Vladimir Rusinov <vladimir.rusinov@gmail.com>
# SPDX-License-Identifier: Apache-2.0
"""
Saving Taxes.

https://www.codechef.com/practice/course/basic-programming-concepts/DIFF500/problems/TAXSAVING
"""

t = int(input())

for _ in range(t):
    case = input().split()
    x = int(case[0])
    y = int(case[1])
    print(x - y)
